cinema/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
```

Comment:
Consider using `CharField` instead of `TextField` for the `description` field if you plan to set a `max_length` constraint. According to the checklist, if you specify `max_length`, it's more reasonable to use `CharField`.